src/picongpu/include/plugins/adios/ADIOSWriter.hpp

```diff
@@ -479,7 +479,7 @@ class ADIOSWriter : public ILightweightPlugin
          */
         ADIOS_CMD(adios_read_init_method(ADIOS_READ_METHOD_BP,
                                          mThreadParams.adiosComm,
-                                         "verbose=3;")); //mpiTransportParams.c_str())); // rly ?
+                                         "verbose=3;abort_on_error;"));
```

Comment:
not that I know of. it should just add some output (but I don't really see more with it). as far as I saw it in the code, it just adds some output behind runtime if's. [Manual `8.4.2  adios_read_init_method`](http://users.nccs.gov/~pnorbert/ADIOS-UsersManual-1.9.0.pdf) says:
```
verbose=<integer>
Set the level of verbosity of ADIOS messages:  0=quiet, 1=errors only, 2=
warnings, 3=info, 4=debug
```